pages/[place]/index.tsx

```
@@ -0,0 +1,90 @@
+interface StaticPathParams {
+  place: string;
+  [key: string]: string | string[] | undefined;
+}
+
+export default function Place({ initialState, placeTypeLabel }: PlaceProps): JSX.Element {
+  // If initialState or events is missing, show a fallback UI
+  if (!initialState || !initialState.events) {
+    return <div>No events found or data is loading.</div>;
+  }
+
+  useEffect(() => {
```

ESLint

[error] React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)